### PR TITLE
Update to 43.0

### PR DESCRIPTION
**Pull request by A6GibKm**

### CODE REVIEW

File: org.gnome.clocks.json [hunk 1]
```diff
@@ -34,8 +34,8 @@
             "sources": [
                 {
                     "type": "archive",
```
Comment (bilelmoussaoui) [inline org.gnome.clocks.json:36]: Make sure you copy the correct config-options btw

Comment (A6GibKm) [reply org.gnome.clocks.json:42]: Nice catch updated the rest.

**Comment by bilelmoussaoui** [comment]
bot, build

**Comment by A6GibKm** [comment]
tested and works.

**Pull request merged**